### PR TITLE
Bring in the latest http_throttle.

### PR DESCRIPTION
**Pull request by nex3**
See dart-lang/pub#1610

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -125,14 +128,24 @@ entirely to allow inference to fill in the type.
...
+
+  * `pub publish`
+  
+    * Added support for the UNLICENSE file.
+
+    * Packages that depend on the Flutter SDK may be published.
+
+  * `pub get` and `pub upgrade`
+
     * Better handling for network errors when fetching packages. These are no
```
Comment (kevmoo) [inline CHANGELOG.md]: While you're here, the 2nd sentence is ... rough...

Comment (nex3) [inline CHANGELOG.md]: Done.

**Review by kevmoo — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -85,7 +85,10 @@ entirely to allow inference to fill in the type.
 ### Tool Changes
 
 * Pub
-    * Added support for the Dart Development Compiler in `build` and `serve`.
```
Comment (kevmoo) [inline CHANGELOG.md]: Sorry...

Does this cleanup of the changelog include behavior changes from the CL?

If yes, those should be in the 1.25 section – this patch won't make 1.24

**Review by kevmoo — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (nex3) [inline CHANGELOG.md]: Done.

**Comment by kevmoo** [comment]
Cool to land? Waiting on @munificent ?

**Comment by nex3** [comment]
> Waiting on @munificent ?

Yep.

**Review by munificent — APPROVED** [review]

**Pull request merged**